cmd/wc/main.go

```diff
@@ -15,35 +15,32 @@ func check(e error) {
 	}
 }
 func main() {
-	args := os.Args[1:]
-	var flagL, flagW, flagC bool = false, false, false
-
-	if slices.Contains(args, "-l") {
-		flagL = true
-	}
-	if slices.Contains(args, "-w") {
-		flagW = true
-	}
-	if slices.Contains(args, "-c") {
-		flagC = true
-	}
+	var flagL bool
+	flag.BoolVar(&flagL, "l", false, "a bool for output lines")
+	var flagW bool
+	flag.BoolVar(&flagW, "w", false, "a bool for output lines")
+	var flagC bool
+	flag.BoolVar(&flagC, "c", false, "a bool for output lines")
+	flag.Parse()
+	filepath := flag.Args()[0]
 	var s []string
-	data, err := os.ReadFile(args[len(args)-1])
+	data, err := os.ReadFile(filepath)
 	check(err)
 	s = strings.Split(string(data), "\n")
 	l := len(s)
-	if flagL {
-		fmt.Println("lines : ", l)
+	if flagL || (!flagL && !flagC && !flagW) {
```

Comment:
you can add a separate check for the other part, if (!flagL && !flagC && !flagW) print the 3 values, no need to test for it 3 times